tenable/io/v3/base/schema/explore/search.py

```diff
@@ -1,29 +1,49 @@
 '''
 Base Explore Search Schema
 '''
-from marshmallow import Schema
-from marshmallow import fields as marshm_fields
+from marshmallow import Schema, ValidationError, fields, post_dump, pre_load
 from marshmallow import validate as v
 
-from .filters import FilterSchema
+from tenable.io.v3.base.schema.explore.filters import FilterSchema
 
 
 class SortSchema(Schema):
     '''
     Schema for the sorting sub-object
     '''
-    property = marshm_fields.Str()
-    order = marshm_fields.Str(validate=v.OneOf(['asc', 'desc']))
+    property = fields.Str()
+    order = fields.Str(validate=v.OneOf(['asc', 'desc']))
+
+    @pre_load(pass_many=True)
+    def transform_data(self, data, **kwargs):
+        if isinstance(data, tuple) and len(data) == 2:
+            property = data[0]
+            order = data[1]
+            return dict(property=property, order=order)
+        elif isinstance(data, dict):
+            return data
```

Comment:
This should be unnecessary, as a validation error would be thrown by marshmallow specific to the instance type